content/guides/dogstatsd.md

```diff
@@ -298,7 +298,7 @@ the packets:
 #### Fields
 - Mandatory:
   - `title` — Event title.
-  - `text` — Event text. Supports line breaks.
+  - `text` — Event text. Insert line breaks with an escaped slash (`\\n`)
```

Comment:
@kmshultz @technovangelist people can send multiple DogStatsD command in one UDP packet if they are split by line break. It's part of the StatsD protocol upon which DogStatsD is based.

For example instead of doing:
`some.metric.a:0.5|g`

then:
`some.metric.b:0.5|g`

People can do
`some.metric.a:0.5|g\nsome.metric.b:0.5|g`

see here: https://github.com/etsy/statsd/blob/master/docs/metric_types.md#multi-metric-packets